packages/client/src/config/autofill.test.ts

```diff
@@ -11,8 +19,94 @@ const hashFromFile = async (filename: string) =>
     .digest("hex");
 
 describe("Autofill", () => {
-  test("autofillResolvers", () => {
-    // @TODO implement tests for autofill
+  describe("works on checker", () => {
+    test("with address 1", () => {
+      const checker = getChecker(mockedChecker1);
+      expect(checker.stack).toEqual([]);
+      expect(checker._getAllQuestions()[0].answer).toEqual(undefined);
+
+      checker.autofill(autofillResolvers, { address: addressMock1 });
```

Comment:
Good question. I used `.autofill()` to test the autofill-resolvers, because that was the most easiest way to get the coverage. I will check if I can update this, this sprint.

Reply:
I've refactored `autofill.ts` so I can import the specific answers into my tests. The tests have been improved and are finished now.